packages/devextreme/testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -4737,37 +4737,79 @@ QUnit.module('reset', () => {
         assert.strictEqual(summaryItemsAfterValidate.length, 2, 'form has validation summary after validation');
     });
 
+    let screen;
```

Comment:
please create a separate module and move this stuff to `beforeEach`. It's really sloppy to me to store variable somewhere between tests. We actually don't wanna to have an access to these variables from other tests